src/explore-education-statistics-admin/src/pages/release/data/components/ApiDataSetMappableTable.tsx

```diff
@@ -73,7 +73,7 @@ export default function ApiDataSetMappableTable({
   return (
     <table
       className="dfe-table--vertical-align-middle dfe-table--row-highlights"
-      id={`mappable-filter-options-${kebabCase(groupKey)}`}
+      id={`mappable-table-${camelCase(groupKey)}`}
```

Comment:
Why the change from `kebabCase` to `camelCase` here? In the styleguide the preference is for kebab.

Reply:
I did this for consistency with location level keys (which are camelCased).

However, it's not really a big deal to go with kebab-casing so I'll switch back.